src/clients/api/mutations/supplyBnb.ts

```diff
@@ -0,0 +1,32 @@
+import Web3 from 'web3';
+import { TransactionReceipt } from 'web3-core';
+import { getVBepToken } from 'utilities';
+
+export interface ISupplyBnbInput {
+  tokenContract: $TSFixMe; // @TODO: use contract type once defined (Typechain?)
+  web3: Web3;
+  account: string | undefined;
+  amount: string;
+}
+
+const vBnbAddress = getVBepToken('bnb').address;
+
+export type SupplyBnbOutput = TransactionReceipt;
+
+const supplyBnb = async ({
+  web3,
+  tokenContract,
+  account,
+  amount,
+}: ISupplyBnbInput): Promise<SupplyBnbOutput> => {
+  const contractData = tokenContract.methods.mint().encodeABI();
```

Comment:
You can use `getTokenContract` to get an instance of the contract. It will be strongly typed and you can call its methods directly:
```
await tokenContract.methods.mint(account, vBnbAddress, amount.toFixed()).send({ from: account });
```